test/functional/data.spec.js

```diff
@@ -2,8 +2,10 @@ const got = require('got');
 const {expect} = require('chai');
 const parallel = require('mocha.parallel');
 const sharp = require('sharp');
+const Ajv = require('ajv');
 
 const data = require('../../data.json');
```

Comment:
I'd say this require should go into a test, so tests wont break if json is semantically not valid

Reply:
Didn't get this point. Why would test break with a require statement?

Reply:
@aslafy-z I think we decided to require it outside because if the json is not valid the test should fail?